pep-0692.rst

```diff
@@ -39,9 +42,66 @@ type annotating ``**kwargs`` is not possible. This is especially a problem for
 already existing codebases where the need of refactoring the code in order to
 introduce proper type annotations may be considered not worth the effort. This
 in turn prevents the project from getting all of the benefits that type hinting
-can provide. As a consequence, there has been a `lot of discussion <mypyIssue4441_>`__
-around supporting more precise ``**kwargs`` typing and it became a
-feature that would be valuable for a large part of the Python community.
+can provide.
+
+Moreover, one pattern that ``**kwargs`` are especially useful for is when a
+function should accommodate optional keyword-only arguments that don't have
+default values. A need for a pattern like that can arise when values that are
```

Comment:
I find this a particularly weak argument -- optional keyword args without defaults seems an obscure edge case, and marker objects usually work just fine. Even if there are real-world examples of this need (as you show below), I don't think it's worth nearly 50 lines of text in the PEP.

Reply:
For what its worth, as a random datapoint from a random Python user, I don't think I've ever seen this pattern in real life—I'm sure it happens, but "especially useful" seems like it might be overstating it and this might be more helpful nearer the end of the section, particularly considering the utility of the other two (I see them absolutely everywhere).

Reply:
That makes sense, I deleted the detailed example and just linked the httpx issue. Also, moved the paragraphs around and changed the "especially useful" bit to "also convenient".